test/functorch/test_ops.py

```diff
@@ -956,6 +955,7 @@ def vjp_of_vjp(*args_and_cotangents):
              {torch.float32: tol(atol=5e-04, rtol=1e-04)}, device_type="cuda"),
     ))
     @skipOps('TestOperators', 'test_vmapvjp', vmapvjp_fail.union({
+        xfail('as_strided'),
```

Comment:
We are just moving above xfail here? Is that correct?

Reply:
Yes, that's correct. Before the xfail was in `vmapvjp_fail`, which is used in other places of this file. With the addition of `index_add`, there is a batch rule for `as_strided`, but it fails on some specific cases.